### PR TITLE
License docker

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ receive pointcloud data from the sonair-evk; and in particular it should serve
 as an example to build upon if you wish to integrate the sonair-evk pointcloud
 into an internal system. In order to visualize the pointcloud from a standalone
 sonair-evk unit you can just use FoxGlove endpoint as described in the
-sonair-evk instructions for use[....].
+sonair-evk Instructions for Use document.
```
Comment (endresonair) [inline README.md]: Add url?

File: README.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ receive pointcloud data from the sonair-evk; and in particular it should serve
...
 
 ## The protocol
 The sonair-evk unit will publish the pointcloud, along with a status flag as
 [CoAP](https://en.wikipedia.org/wiki/Constrained_Application_Protocol) messages
 over UDP. The IP address for these packages is configured by the user with
-sonair configuration tool [....]. When used with a ROS publishing node we
+sonair configuration tool. When used with a ROS publishing node we
```
Comment (endresonair) [inline README.md]: on the Sonair Evaluation Kit Configuration page.

File: README.md [hunk 3]
```diff
@@ -78,6 +78,6 @@ which can be used as an example.
 ```
 bash% cd ros
 bash% docker build -t ros-sonair-evk -f Dockerfile.sonair-evk .
-bash% docker run -p 5683:5683/udp -p 8765:8765 --rm -it ros-sonair-evk
+bash% docker run -p 5683:5683/udp -p 8765:8765 --rm ros-sonair-evk
```
Comment (endresonair) [inline README.md:81]: Add `./start_ros.sh` at the end of docker run

File: ros/Dockerfile.sonair-evk [hunk 4]
```diff
@@ -17,4 +17,4 @@ COPY sonair_evk/start_ros.sh .
 RUN . /opt/ros/iron/setup.sh && colcon build --symlink-install --packages-select sonair_evk
 RUN . install/local_setup.sh
 RUN echo ". /sonair/ros/install/local_setup.bash" >> "/root/.bashrc"
-
+ENTRYPOINT ["/sonair/ros/start_ros.sh"]
```
Comment (endresonair) [inline ros/Dockerfile.sonair-evk:20]: Ah, ok, then you can disregard the comment above..

File: ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py [hunk 5]
```diff
@@ -1,4 +1,6 @@
-import struct
+# Copyright (C), 2024, Sonair AS
+#
+# This software is free software, licensed according to the MIT License:https://en.wikipedia.org/wiki/MIT_License import struct
```
Comment (endresonair) [inline ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py:3]: Ah, nice, add this to there rest of the files as well?

File: ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py [hunk 6]
```diff
@@ -1,4 +1,6 @@
...
 from typing import Tuple
```
Comment (endresonair) [inline ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py:4]: It is common to include a license file in the repo as well I think? See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository

**Review by endresonair — APPROVED** [review]

File: ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py (hunk 5 above)
Comment (joakim-hove) [reply ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py:3]: My thinking was that the other files did not really contain anything close to Sonair IP - and so really should not need such license information. I'll merge merge this now - but if you think a license term should be added to all the files I will add that in a subsequent PR.

**Pull request merged**